tests/test_class.py

```diff
@@ -31,6 +31,12 @@ def test_instance_new(msg):
     assert cstats.alive() == 0
 
 
+def test_classmethod(num_instances=10):
+    for i in range(num_instances):
+        assert getattr(m.NoConstructor, "uuid", 0) == i
+        m.NoConstructor.new_instance_uuid()
+
```

Comment:
Maybe (untested):
```
    assert no hasattr(m.NoConstructor, "uuid")
    for i in range(num_instances):
        m.NoConstructor.new_instance_uuid()
        assert m.NoConstructor.uuid == i + 1
```

Reply:
Oh ...
Commit 2e3d29d from two months ago escaped my attention.
If you reply with Done or similar here it's more likely that that doesn't happen.